src/commands/service/create.ts

```diff
@@ -116,12 +142,20 @@ export const execute = async (
 
   if (serviceName === "." && opts.displayName === "") {
     logger.error(
-      `If specifying the current directory as service name, please incluce a display name using '-n'`
+      `If specifying the current directory as service name, please include a display name using '-n'`
     );
     await exitFn(1);
     return;
   }
 
+  // validate user inputs are DNS compliant
+  try {
+    assertValidDnsInputs(opts);
+  } catch (err) {
+    logger.error(err);
+    await exitFn(1);
+  }
```

Comment:
please add a `return;` after the `await exitFn(1);`. otherwise the code after this will continue to run because there is a time gap before `process.exit(1)` is executed. This is due to the wait time for winston.logger to flush buffer to log file

Reply:
take it back. it should be ok.